openc3/lib/openc3/interfaces/interface.rb

```diff
@@ -239,7 +245,12 @@ def read
         end
         next if data == :STOP
 
-        packet = convert_data_to_packet(data)
+        # Extra check is for backwards compatibility
+        if extra
+          packet = convert_data_to_packet(data, extra)
+        else
+          packet = convert_data_to_packet(data)
+        end
```

Comment:
convert_data_to_packet simply does `Packet.new(nil, nil, :BIG_ENDIAN, nil, data)`. Why does it need extra?

Reply:
Will fix.